source/jormungandr/jormungandr/scenarios/tests/new_default_tests.py

```diff
@@ -247,3 +248,89 @@ def merge_responses_on_errors_test():
     # both messages must be in the composite error
     assert resp1.error.message in merged_response.error.message
     assert resp2.error.message in merged_response.error.message
+
+
+def add_pt_sections(journey):
+    section = journey.sections.add()
+    section.type = response_pb2.STREET_NETWORK
+    section.street_network.mode = response_pb2.Walking
+    section = journey.sections.add()
+    section.type = response_pb2.PUBLIC_TRANSPORT
+    section = journey.sections.add()
+    section.type = response_pb2.STREET_NETWORK
+    section.street_network.mode = response_pb2.Walking
+
+
+def create_next_kraken_request_test():
+    request_clock = {}
```

Comment:
minor: but if you want to win some lines you can do:
`request_clock = {'datetime':100000, 'clockwise':True }`